cookbooks/elasticsearch/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
 default['elasticsearch'].tap do |elasticsearch|
   # Run Elasticsearch on util instances named elasticsearch_*
   # This is the default
-  elasticsearch['is_elasticsearch_instance'] = ( node['dna']['instance_role'] == 'util' && node['dna']['name'].include?('elasticsearch_') )
+  #elasticsearch['is_elasticsearch_instance'] = ( node['dna']['instance_role'] == 'util' && node['dna']['name'].include?('elasticsearch_') )
 
   # Run Elasticsearch on a solo or app_master instance
   # Not recommended for production environments
-  #elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
+  elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
```

Comment:
Leftover from my testing. I should revert this to the previous default which is to install on a named utility instance